weave/panels/panel_group.py

```diff
@@ -74,6 +74,9 @@ class GroupConfig(typing.Generic[ItemsType]):
     enableAddPanel: typing.Optional[bool] = dataclasses.field(
         default_factory=lambda: None
     )
+    enableDeletePanel: typing.Optional[bool] = dataclasses.field(
```

Comment:
optional nit: having a parameter be treated as `true` when it does not exist feels counterintuitive. I might change this to `disableDeletePanel` since that is more aligned what the caller is trying to do. This then means code can do `if disableDeletePanel` or `if not disableDeletePanel` rather than `if enableDeletePanel == None or enableDeletePanel`